.github/scripts/automergeCrowdinPr.js

```diff
@@ -38,41 +49,46 @@ module.exports = async ({ github, context }) => {
     return
   }
 
-  console.log(`Verifying that only expected files are modified for PR #${pr.number}`)
-  const listFiles = await github.rest.pulls.listFiles({
+  console.log(`Fetching reviews for ${pr.number}`)
```

Comment:
i couldn't resist refactoring this part (there aren't many changes if viewing this PR with whitespaces ignored)